keel-core/src/main/kotlin/com/netflix/spinnaker/keel/artifacts/ArtifactVersionLinks.kt

```diff
@@ -42,6 +44,6 @@ fun getScmBaseLink(scmInfo: ScmInfo, commitLink: String): String? {
     "stash" in commitLink ->
       return scmBaseURLs["stash"]
     else ->
-      throw UnsupportedScmType(message = "Stash is currently the only supported SCM type")
+      throw UnsupportedScmType(message = "Stash and Git are currently the only supported SCM types")
```

Comment:
I don't think you need to change this because you'll never get here if the SCM is GitHub.

Reply:
true true. left it more as a hint for people to know what they can use in case it is neither git nor stash. can also get rid of it.

Reply:
I'm fine with keeping it. Just suggested updating the message to be more accurate.